config/deploy.rb

```diff
@@ -0,0 +1,54 @@
+# config valid only for current version of Capistrano
+lock '3.4.0'
+
+set :application, 'timeoverflow'
+set :repo_url, 'git@github.com:coopdevs/timeoverflow.git'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.